views/misc-privacy.html

```diff
@@ -0,0 +1,27 @@
+<div class="dialog_popup callflows-port">
+	<h1>{{ i18n.oldCallflows.privacy.title }}</h1>
+	<form name="form" method="post" action="#">
+		<div class="form_content">
+			<div class="popup_field">
+				<label for="privacy_mode">{{ i18n.oldCallflows.privacy.mode }}: </label>
+				<select id="privacy_mode" style="width: 100px; height: 28px;">
```

Comment:
Please, no styling in the template. This should be done in the dedicated `/submodules/misc/misc.scss` style sheet.